### PR TITLE
fix(integration test): upgrade localstack

### PR DESCRIPTION
**Pull request by STRRL**
## Related changes

- [ ] This change also requires further updates to the [website](https://github.com/chaos-mesh/website) (e.g. docs)
- [ ] This change also requires further updates to the `UI interface`

## Cherry-pick to release branches (optional)

> This PR should be cherry-picked to the following release branches:

- [x] release-2.6
- [x] release-2.5

### CHANGELOG

> Must include at least one of them.

- [ ] I have updated the `CHANGELOG.md`
- [x] I have labeled this PR with "no-need-update-changelog"

### Tests

> Must include at least one of them.

- [ ] Unit test
- [ ] E2E test
- [x] Manual test

### Side effects

- [ ] **Breaking backward compatibility**

## DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by STRRL** [comment]
this PR fix the broken integration tests, PTAL @g1eny0ung

File: .github/workflows/integration_test.yml [hunk 1]
```diff
@@ -106,8 +106,8 @@ jobs:
           make chaosctl
       - name: Install localstack && aws client
         run: |
-          helm repo add localstack-repo http://helm.localstack.cloud
-          helm upgrade --install localstack localstack-repo/localstack --version 0.1.2
+          helm repo add localstack-repo http://helm.localstack.cloud && helm repo update
```
Comment (g1eny0ung) [inline .github/workflows/integration_test.yml:109]: Should we need to run `helm repo update`? The `helm repo add` command should already get the latest version. 🤔

Comment (STRRL) [reply .github/workflows/integration_test.yml:109]: Yes, you're right, we do not need `helm repo update` for environments in github actions.

**Comment by STRRL** [comment]
PTAL @g1eny0ung

**Pull request merged**